### ISSUE TITLE
Add more badges

### ISSUE THREAD

**Issue by mfeurer** [openml/openml-python#844]
* Example: https://github.com/py4ds/nbless
* Another one: https://www.repostatus.org/

**Comment by mfeurer** [comment]
Or we should remove them again because we now use github actions instead of travis-ci? @PGijsbers what do you think right now?

**Comment by PGijsbers** [comment]
I imagine there are also badges for Github CI? I don't think more badges is necessarily better, too much clutter is also a thing. I think the main thing we should ask is what function the badges should serve. I'd say they are primarily for people looking into first time contributing or using the package. So I'd get rid of `develop` coverage and test status (developers will see them on their PR, users don't use `develop`). I think the linked `nbless` page is a good example of showing only relevant information to the target audience.

**Comment by mfeurer** [comment]
> I think the main thing we should ask is what function the badges should serve.

Yes, I agree with your statement, it's most likely serves as a standardized list of the features supported by the packages. What worries me most about having these badges is that we can't get the builds stable, so this will always look bad for us...

**Comment by PGijsbers** [comment]
I agree it doesn't have the best look, but at the same time it's unfortunately just the truth that there's work left when it comes to improving our tests/infrastructure.

**Comment by mfeurer** [comment]
TODO: we also need to re-add badges for regular CI as we moved from travis-ci to github actions.
